src/components/Body.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+import Main from './Main';
+
+const AppBody = styled.div`
```

Comment:
The order of functions DOSE matter in JS as you know but the order of Components in React JS does not in one file.

But you better have the main function above and then put all the styled components

Reply:
Done